FrontEnd/src/components/admin/CreateOrEditUserForm.js

```diff
@@ -1,51 +1,69 @@
-import generalStyles from "./../../styles/GeneralStyles"
-import {Typography} from "@material-ui/core";
-import Paper from "@material-ui/core/Paper";
-import TextField from "@material-ui/core/TextField";
-import IconButton from "@material-ui/core/IconButton";
-import NoteAddIcon from "@material-ui/icons/NoteAdd";
+import {generalNativeStyles} from "../../styles/GeneralStyles"
+import { Button, Card, Input, Text} from "react-native-elements"
 import {SERVER} from "../../../index";
 import { USER_ADMINISTRATION_IDENTIFIER} from "../general/identifiers";
 import {switchContent} from "../../actions/teacherNavigationActions";
-import makeStyles from "@material-ui/core/styles/makeStyles";
-import FormControl from "@material-ui/core/FormControl";
-import InputLabel from "@material-ui/core/InputLabel";
-import Select from "@material-ui/core/Select";
-import MenuItem from "@material-ui/core/MenuItem";
+import {CheckBox} from "react-native-elements";
 
-const customStyles = makeStyles((theme) => ({
-    backgroundPaper: {
-        width: '100%',
-        maxWidth: '800px',
+const customstyles = StyleSheet.create({
+    checkBoxView: {
         display: 'flex',
-        flexDirection: 'column',
-        padding: theme.spacing(2),
-        justifyContent: "center",
+        flexDirection: 'row',
+        justifyContent: 'center',
+        flexWrap: 'wrap',
     },
-    formTextfield: {
-        marginTop: '2%',
+    checkBoxContainerStyle: {
+        backgroundColor: "#ffffff",
+        borderWidth: 0,
     },
-    formControl: {
-        marginTop: '2%',
-    },
-}))
+    cardStyle: {
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+    }
+})
 
 function CreateOrEditUserForm(props) {
-    const generalStyle = generalStyles();
-    const customStyle = customStyles();
     const [firstname, setFirstname] = useState("");
     const [lastname, setLastname] = useState("");
     const [role, setRole] = useState("student");
-    const [firstnameFieldFilled, setFirstnameFieldFilled] =useState(true);
-    const [lastnameFieldFilled, setLastnameFieldFilled] =useState(true);
+    const [firstnameFieldFilled, setFirstnameFieldFilled] = useState(true);
+    const [lastnameFieldFilled, setLastnameFieldFilled] = useState(true);
+    const [studentChecked, setStudentChecked] = useState(true);
+    const [teacherChecked, setTeacherChecked] = useState(false);
+    const [adminChecked, setAdminChecked] = useState(false);
 
-    function handleFormChange(event, type) {
+    function handleFormChange(value, type) {
         switch (type) {
-            case "firstname": setFirstname(event.target.value); break;
-            case "lastname": setLastname(event.target.value); break;
-            case "role": setRole(event.target.value);
+            case "firstname": setFirstname(value); break;
+            case "lastname": setLastname(value); break;
+        }
+    }
+    const handleRoleSelectionChange = (role) => {
```

Comment:
What the heck?
setStudentChecked(role === "student");
setTeacherChecked(role === "teacher");
setAdminChecked(role === "admin");